### DISCUSSION TITLE
Is it possible to customize the Memory widget in this way?

### DISCUSSION THREAD

**Discussion by lemerchand** [ClementTsang/bottom#602] [Q&A] [answered]
I tend to have a Bottom open in a small terminal tiled among other windows and it gets a bit scrunched. I'm not really concerned with the Memory widget's graph and I was hoping it was possible to reduce it to just the text-based usage (viz., "RAM: X% Y gb / Z gb")?

Does anyone know if it's possible and what I would do to go about it?

**Comment by ClementTsang** [comment, marked as answer]
Currently not possible but I can look into adding a non-graph widget as an option.  Not hard to add.

**Reply by lemerchand** [reply]
Wow--What a fast response. 

> Currently not possible but I can look into adding a non-graph widget as an option. Not hard to add.

That would be super awesome!  I think I could really get the layout where I want it if that were the case. 

Thanks!
